graphicsDatatypes.h

```diff
@@ -0,0 +1,34 @@
+#ifndef GRAPHICSDATATYPES_H_
+#define GRAPHICSDATATYPES_H_
+
+typedef struct{
+    uint8_t reserved;
+
+    uint8_t redValue;
+    uint8_t greenValue;
+    uint8_t blueValue;
+}colorStruct;
```

Comment:
- in graphics.h verschieben
- tag hinzufügen (`typedef struct tag_tsGraphicsColor {...} tsGraphicsColor`)
- Umbenennen in `tsColor` entsprechend Coding Style
- Packing festlegen (`__attribute__((packed))`)

Reply:
wird in anderen files als datentyp benötigt. Was ist der tag?

was ist packing?

Reply:
Dann in graphics.h tun. Das Text-Modul (text.c) muss dann graphics.h einbinden, auch wenn nur diese Struktur benutzt wird und sonst nix